### PR TITLE
chore: update eval warning msg

### PR DESCRIPTION
**Pull request by underfin**
### Description

Here port rollup eval code and msg, the pervious msg is not clear.

### CODE REVIEW

**Review by hyf0 — DISMISSED** [review]

File: crates/rolldown_error/src/events/eval.rs [hunk 1]
```diff
@@ -7,36 +7,36 @@ use crate::{diagnostic::Diagnostic, types::diagnostic_options::DiagnosticOptions
...
-    diagnostic.title = "Rolldown does not support `eval` function currently.".to_string();
+    diagnostic.title = "Use of eval is strongly discouraged as it poses security risks and may cause issues with minification.".to_string();
 
     let file_id = diagnostic.add_file(filename, Arc::clone(&self.source));
 
     diagnostic.add_label(
       &file_id,
       self.eval_span.start..self.eval_span.end,
-      "Used `eval` function here.".to_string(),
+      "Used `eval` function at here.".to_string(),
```
Comment (Boshen) [inline crates/rolldown_error/src/events/eval.rs]: `at here`? Can we align the english sentence to similar tools?

Comment (underfin) [reply crates/rolldown_error/src/events/eval.rs]: Not for sure. What is better for this?

Comment (Boshen) [reply crates/rolldown_error/src/events/eval.rs]: "Used `eval` function here." was correct, but I suggest aligning with other tools, if they throw one.

**Review by hyf0 — APPROVED** [review]

File: crates/rolldown_error/src/events/eval.rs (hunk 1 above)
Comment (underfin) [reply crates/rolldown_error/src/events/eval.rs]: Not find similar msg at rollup, just revert it.

**Pull request merged**